trackers/javascript-tracker/src/in_queue.ts

```diff
@@ -265,6 +265,11 @@ export function InQueueManager(functionName: string, asyncQueue: Array<unknown>)
             // Strip GlobalSnowplowNamespace from ID
             fnTrackers[tracker.id.replace(`${functionName}_`, '')] = tracker;
           }
+
+          // Append the `fnTrackers` object to the parameterArray, to allow it to be
+          // accessed as the final argument in the callback, useful in environments that don't support `this` (GTM)
+          Array.prototype.push.call(parameterArray, fnTrackers);
```

Comment:
Nitpick, but might be safer to create a new concatenated array (`parameterArray + [fnTrackers]`) rather than mutating `parameterArray`? I see `parameterArray` being used below although likely not in the case this is function, but I guess better to be safe.

Reply:
Fine by me! Had to do it in a bit less concise way than that though 😄